src/schema/v2/__tests__/recentlySoldArtworks.test.ts

```diff
@@ -0,0 +1,77 @@
+import { StaticPathLoader } from "lib/loaders/api/loader_interface"
+import { runQuery } from "schema/v2/test/utils"
+
+describe("RecentlySoldArtworks", () => {
+  const artworksLoader: StaticPathLoader<any> = (params) => {
+    const ids = params && params.ids ? params.ids : []
+    return Promise.resolve(
+      ids.map((id) => ({
+        _id: id,
+      }))
+    )
+  }
+
+  it("returns all artworks", async () => {
+    const query = `
+      {
+        recentlySoldArtworks {
+          edges {
+            node {
+              artwork {
+                internalID
+              }
+              lowEstimateUSD
+              highEstimateUSD
+              priceRealized
+            }
+          }
+        }
+      }
+      `
+
+    const context = {
+      artworksLoader,
+    }
+
+    const response = await runQuery(query, context)
+    const recentlySoldArtworks = response.recentlySoldArtworks
+
+    expect(recentlySoldArtworks.edges).toHaveLength(20)
```

Comment:
**suggestion:** If you change `20` to `recentlySoldArtworks.length`, this test assertion will make more sense to future developers.